parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -253,28 +253,17 @@ configureCNIIPTables() {
     fi
 }
 
-ensureContainerRuntime() {
-    if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
-        ensureDocker
-    fi
-    {{if NeedsContainerd}}
-        ensureContainerd
-    {{end}}
-}
-
 {{if NeedsContainerd}}
 ensureContainerd() {
   wait_for_file 1200 1 /etc/systemd/system/containerd.service.d/exec_start.conf || exit $ERR_FILE_WATCH_TIMEOUT
   wait_for_file 1200 1 /etc/containerd/config.toml || exit $ERR_FILE_WATCH_TIMEOUT
-  {{if IsKubenet }}
```

Comment:
why do we remove this ?

Reply:
docker/moby-engine sets up the necessary `ip_forwarding=1` kernel rules to forward pod traffic to the cbr0/azure0 bridge. The new containerd VHDs don't have docker/moby-engine installed so we need to set this up ourselves.

Reply:
i'm just curious  - why did we have to do this for kubenet only, but no azurecni, when moby-engine is present?

Reply:
I believe this was a mistake on my part. The sysctl conf file here was technically never needed when we were using VHDs that had moby-engine. The file is definitely needed right now though.

Reply:
TODO: separate PR for cse